examples/concurrency/lib/simple_isolate_closure.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:isolate';
+
+const String filename = 'with_keys.json';
+
+// #docregion main
+void main() async {
+  // Read some data.
+  final jsonData = await Isolate.run(() {
+    final fileData = await File(filename).readAsString();
+    final jsonData = jsonDecode(fileData) as Map<String, dynamic>; 
+    return await jsonData;
```

Comment:
This `await` is unnecessary and should be removed.

(The code from `()` to `}` should be precisely the same as in `_readAndParseJason` below.)